frontend/www/logout.php

```diff
@@ -1,9 +1,22 @@
 <?php
+namespace OmegaUp;
+require_once(dirname(__DIR__, 1) . '/server/bootstrap.php');
 
-require_once('../server/bootstrap.php');
 if (\OmegaUp\Controllers\Session::currentSessionAvailable()) {
     \OmegaUp\Controllers\Session::unregisterSession();
 }
 
-require_once('../server/bootstrap_smarty.php');
-$smarty->display('../templates/logout.tpl');
+\OmegaUp\UITools::render(
+    function (\OmegaUp\Request $r): array {
+        $scripts = [];  // por consistencia con el nombre en L16.
```

Comment:
```suggestion
        $scripts = [];
```

ah ese era un comentario para tí >< normalmente todo el código y los comentarios van en inglés, así que si ves algo en español, la intención es no agregarlo al código, sino explicación de por qué la sugerencia.

Reply:
Perdon se me fue

Reply:
no lo vi